modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotPartitionsVerifyHandler.java

```diff
@@ -362,6 +362,9 @@ private Map<PartitionKeyV2, PartitionHashRecordV2> checkDumpFiles(
         SnapshotHandlerContext opCtx,
         Set<File> partFiles
     ) {
+        if (F.isEmpty(partFiles))
```

Comment:
It looks like this condition should be outside of the method. If param is empty then no need to check them?

Reply:
Fixed but works wierd. Snapshot contains meta. It is a partition but might be ignored too